proto/cosmos/gov/v1/gov.proto

```diff
@@ -195,17 +228,85 @@ message TallyParams {
   // Minimum proportion of Yes votes for proposal to pass. Default value: 0.5.
   string threshold = 2 [(cosmos_proto.scalar) = "cosmos.Dec"];
 
-  // Minimum value of Veto votes to Total votes ratio for proposal to be
-  // vetoed. Default value: 1/3.
-  string veto_threshold = 3 [(cosmos_proto.scalar) = "cosmos.Dec"];
+  reserved 3; // veto_threshold
+
+  // quorum for constitution amendment proposals
+  string constitution_amendment_quorum = 4 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // Minimum proportion of Yes votes for a Constitution Amendment proposal to pass. Default value: 0.9.
+  string constitution_amendment_threshold = 5 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // quorum for law proposals
+  string law_quorum = 6 [(cosmos_proto.scalar) = "cosmos.Dec"];
```

Comment:
shouldn't we add the deprecated flag? not needed/irrelevant?

Reply:
ditto: https://github.com/atomone-hub/cosmos-sdk/pull/17/files/92059e4c83765ae9f6afa8d6e37748f2cd5e8351#r2577190613